authentik/api/decorators.py

```diff
@@ -1,6 +1,6 @@
 """API Decorators"""
+from collections.abc import Callable
```

Comment:
TIL that typing.Callable is deprecated https://docs.python.org/3/library/typing.html#typing.Callable